tests/integration/test_retry_aiohttp.py

```diff
@@ -5,17 +5,27 @@
 from uplink.clients import io
 
 from . import test_retry
+import sys
 
 
 @pytest.mark.asyncio
 async def test_retry_with_asyncio(mock_client, mock_response):
-    import asyncio
+    # Check the Python version and define coroutine accordingly
+    if sys.version_info >= (3, 5):
+        # For Python 3.5 and newer
```

Comment:
non-blocking: I would expect this to throw a SyntaxError for Python 2.7. But, we are way beyond EOL now, and should drop 2.7 support IMO.